src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx

```diff
@@ -110,13 +110,14 @@ export default class SendBroadcastForm extends React.Component<
         <p />
         <TextInputElement
           name={i18n.t('forms.message', 'Message')}
-          showLabel={false}
+          showLabel={true}
           count={Count.SMS}
           onChange={this.handleMessageUpdate}
           entry={this.state.message}
           autocomplete={true}
           focus={true}
           textarea={true}
+          maxLength={150}
```

Comment:
All new defined max lengths should not exist, Flows does not limit the message length like that
The design should be modified to remove that from screens

Reply:
Ok, I just removed it!